python/tvm/relay/backend/contrib/ethosu/tir_to_cs_translator.py

```diff
@@ -617,6 +621,21 @@ def _create_npu_resampling_mode(
     return mode_map[mode]
 
 
+def _create_npu_rounding_mode(
+    mode,
+):
```

Comment:
Can you add the argument and the return type hints?

Reply:
woops